pyteal/compiler/optimizer_test.py

```diff
@@ -1,8 +1,7 @@
 from .optimizer import OptimizeOptions, _apply_slot_to_stack
 
-from ... import *
-
-import pytest
+from pyteal import *
+from pyteal.ast import *
```

Comment:
nonrelative import, importing ast not necessary

Reply:
👍